tests/testthat/test-plot_predictions.R

```diff
@@ -74,17 +76,45 @@ test_that("plot_predictions() works without median", {
     is.na(quantile) | quantile != 0.5
   )
 
-  p <- scoringutils::plot_predictions(
-    example3, x = "target_end_date",
-    filter_truth = list("target_end_date > '2021-06-25'",
-                        "target_end_date <= '2021-07-12'"),
-    filter_forecasts = list("model == 'EuroCOVIDhub-ensemble'",
-                            "forecast_date == '2021-07-12'"),
-    facet_formula = location_name ~ target_type
-  )
+  p <- example3 %>%
+    filter_data(what = "truth",
+                target_end_date > '2021-06-25',
+                target_end_date <= '2021-07-12') %>%
+    filter_data(what = "forecast",
+                model == 'EuroCOVIDhub-ensemble',
+                forecast_date == '2021-07-12') %>%
+    plot_predictions(
+      x = "target_end_date",
+      by = c("location_name", "target_type")
+    ) +
+    facet_wrap(location_name ~ target_type, scales = "free_y")
+
   expect_s3_class(p, "ggplot")
 
   skip_on_cran()
   vdiffr::expect_doppelganger('no_median', p)
 
 })
+
+
+test_that("filter_data() works", {
+
+  example_quantile %>%
```

Comment:
Why are these detached statements here? Shouldn't they be linked up to tests. Nice in general to have tests beyond does this error